app.js

```diff
@@ -186,3 +190,27 @@ function syncToTarget (event, target) {
     server.replicateNetwork(socket, 'pull')
   }
 }
+
+function getGlobalDatasetCentroid (done) {
+  var bbox = [[-90,90],[-180,180]]
+  var loc = null
+
+  var stream = osm.queryStream(bbox)
+  stream.on('data', function (doc) {
```

Comment:
Rather than put this into streaming mode, we can call `stream.read()` multiple times until we get what we want. This would avoid issues with closing the stream